ci/cron/wednesday.yml

```
@@ -16,54 +16,97 @@ jobs:
 - job: open_release_pr
   timeoutInMinutes: 60
   pool:
-    name: linux-pool
-    demands: assignment -equals default
+    vmImage: ubuntu-latest
   steps:
   - checkout: self
     persistCredentials: true
-  - bash: ci/dev-env-install.sh
   - bash: |
-      set -euo pipefail
-      eval "$(./dev-env/bin/dade-assist)"
+      set -euxo pipefail
 
-      setvar() {
-          echo "Setting '$1' to '$2'"
-          echo "##vso[task.setvariable variable=$1;isOutput=true]$2"
+      if header=$(git config 'http.https://github.com/digital-asset/daml.extraheader'); then
+          AUTH="$header"
+      else
+          AUTH="Authorization: basic $(git config remote.origin.url | grep -o '://.*:.*@' | cut -c4- | rev | cut -c2- | rev)"
+      fi
+
+      BASE_SHA=$(git rev-parse HEAD)
+      az extension add --name azure-devops
+      echo "$(System.AccessToken)" | az devops login --org "https://dev.azure.com/digitalasset"
+
+      reset() {
+          git checkout -f $BASE_SHA
+          git reset --hard
       }
+      open_pr() {
+          local branch title body out pr_number
+          branch=$1
+          title="$2"
+          body="$3"
+          out=$4
+          git branch -D $branch || true
+          git checkout -b $branch
+          git add .
+          git -c user.name="Azure Pipelines DAML Build" \
+              -c user.email="support@digitalasset.com" \
+              commit \
+              -m "$(printf "$title\n\n$body\n\nCHANGELOG_BEGIN\nCHANGELOG_END\n")"
+          git push origin $branch:$branch
+          pr_number=$(curl -H "Content-Type: application/json" \
+                           -H "$AUTH" \
+                           --silent \
+                           --location \
+                           -d "{\"title\": \"$title\", \"head\": \"$branch\", \"base\": \"master\", \"body\": \"$body\"}" \
+                           https://api.github.com/repos/digital-asset/daml/pulls \
+                      | jq '.number')
+          az pipelines build queue \
+              --branch $branch \
+              --definition-name "digital-asset.daml" \
+              --org "https://dev.azure.com/digitalasset" \
+              --project daml
+          echo $pr_number > $out
```

This doesn't seem to be used anywhere. I might be mistaken.

---

It's used to get the PR number so we can assign the Standard-Change label to it, assign the release handler, and mention it in the second PR message.

---

Aha, it's unused in the second `open_pr`, but not the first. Can I suggest making it an optional parameter, rather than calling `mktemp`?

```suggestion
          if [[ -n "$out" ]]; then
            echo $pr_number > $out
          fi
```